gemhardware/devices/src/common/optohybrid/HwOptoHybrid.cc

```diff
@@ -198,6 +197,15 @@ void gem::hw::optohybrid::HwOptoHybrid::setVFATsToDefaults()
   configureVFATs();
 }
 
+void gem::hw::optohybrid::HwOptoHybrid::configureOHCalDataFormat(uint32_t const mask)
+{
+    CMSGEMOS_DEBUG("HwOptoHybrid::configOHCalDataFormat mask"
+                   << std::hex << std::setw(8) << std::setfill('0') << mask << std::dec);
+    std::stringstream regName;
+    regName << "GEM_AMC.DAQ.CONTROL.CALIBRATION_MODE_CHAN" ;
+    return writeReg(regName.str(), mask&0x00ffffff);
+}
```

Comment:
I'm not sure this captures the register meaning at all. The `CALIBRATION_MODE_CHAN` register selects the VFAT3 input channel (referenced by its index) to be stored in the DAQ data packet. The same setting applies to all the OH connected to a given AMC.